docs/dabp/guides/functional/manage-policy-via-ui.md

```diff
@@ -0,0 +1,29 @@
+# Manage policies
+
+You need to create a policy before you can assign it to groups and members.
+Policies can be created on the "Policies" page or via the [Create Policy API](../technical/create-policy-via-api.md). 
+
+To manage policies, you must be an administrator with the `Add/edit/delete policies` permission on the root group. 
+That is the top-level group. 
```

Comment:
We defined an administrator as a "user with all permissions on the root group", therefore you cannot be an `administrator on the root group`. So we should use `superuser` here.

Reply:
I will change it to  'user' in order to keep it general.